web/src/lib.rs

```diff
@@ -1436,7 +1437,9 @@ impl ExternalInterfaceProvider for JavascriptInterface {
     fn on_callback_available(&self, name: &str) {
         self.js_player.on_callback_available(name);
     }
+}
 
+impl FsCommandProvider for JavascriptInterface {
```

Comment:
We now need a `with_fs_commands` call for this, right?

Reply:
Good catch, fixed!